Tools/generate-docc-reference/GenerateDoccReference.swift

```diff
@@ -71,6 +76,8 @@ struct GenerateDoccReference: ParsableCommand {
 
   func run() throws {
     let data: Data
+    // runs the tool with the --experimental-dump-help argument to capture
```

Comment:
these comments are nice, but the entire body of this file is nearly identical to the version in generate manual... might be worth duplicating them?

Reply:
I didn't even look there, but I'll go review - this was all me commenting to future me about what was happening and in play.